assets/js/components/MiniChart.js

```diff
@@ -72,14 +72,10 @@ function MiniChart( { index, change } ) {
 	return (
 		<div className="googlesitekit-mini-chart">
 			<GoogleChart
+				chartType="PieChart"
 				data={ data }
+				loadingHeight={ 24 }
```

Comment:
Same here, it should be a string:

```suggestion
				loadingHeight="24px"
```